core-build/test-web-library-build/package.json

```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@types/chai": "3.4.34",
     "chai": "~3.5.0",
-    "@types/jest": "~20.0.4",
+    "@types/jest": "~21.1.8",
```

Comment:
>"~21.1.8" [](start = 19, length = 9)

do you need the PNPM workaround in this PR?

Reply:
Yes, included

---
In reply to: [159781970](https://github.com/Microsoft/web-build-tools/pull/466#discussion_r159781970) [](ancestors = 159781970)